zabbix/agent/repo.sls

```diff
@@ -20,6 +20,10 @@ extend:
     pkgrepo:
       - require_in:
         - pkg: zabbix-agent
+    {% elif salt['grains.get']('os_family') == 'Suse' -%}
+    pkgrepo:
+      - require_in:
+        - pkg: zabbix-agent
```

Comment:
```suggestion
```

This current method also splits the `RedHat` block, which will cause problems there.

All this file needs is the line above to be changed:

```diff
-    {% if salt['grains.get']('os_family') == 'Debian' -%}
+    {% if salt['grains.get']('os_family') in ['Debian', 'Suse'] -%}
```

Reply:
Note, this single line change is also required here:

https://github.com/saltstack-formulas/zabbix-formula/blob/c9aeb377d070cae54aa82f15904ab799b5994980/zabbix/server/repo.sls#L15